pkg/db/migration/000011_init.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TYPE CONNECTOR_VALID_CONNECTOR_TYPE RENAME VALUE 'CONNECTOR_TYPE_BLOCKCHAIN' TO 'CONNECTOR_TYPE_APPLICATION';
```

Comment:
@donch1989 I ran the migration command locally and all the references to this type in other tables (i.e. `connector`) were updated successfully.

I'm not aware how we're conducting migrations at this moment in staging / production, let's keep this in mind for the next release.

Reply:
The staging/production use the same command `pipeline-backend-migrate`. So if the Core works, then the cluster should work as well.